tutorial_files/scm_read_obs.py

```diff
@@ -0,0 +1,341 @@
+#!/usr/bin/env python
+
+from netCDF4 import Dataset
+import datetime
+import numpy as np
+import sys
+import math
+import forcing_file_common as ffc
+
+def read_twpice_obs(obs_file, time_slices, date):
```

Comment:
In this file you have a mix of spacings used for indenting. They should be consistent, and I'd suggest making all levels indented 4 spaces.

Reply:
This is a copy of the file in /scm/etc/scripts that has a one-line change for the tutorial. Both versions are officially "unsupported" and are part of the plotting (which sprung up as code that I used for plotting and wasn't necessarily intended to be shared). All of that code needs to be refactored, and we've been limping it along for years now. I'd rather not make a series of small changes in favor of a wholesale rewrite at some point. This was funded in the hurricane supplemental @NCAR (to do SCM initialization, running, plotting inside of a standalone app), but I left before completing that deliverable.